ibm_catalog.json

```diff
@@ -0,0 +1,460 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-cloud-logs",
+        "label": "Cloud automation for Cloud Logs",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "solution",
+          "logging_monitoring"
+        ],
+        "keywords": [
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "cloud logs",
+          "observability",
+          "logging",
+          "monitoring",
+          "monitor",
+          "log"
+        ],
+        "short_description": "Creates and configures IBM Cloud Logs resources",
+        "long_description": "This architecture supports creating and configuring IBM Cloud Logs resources.",
```

Comment:
This needs much more info. for a good example, see https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/blob/66fa626642d0427f3b38a07bbc46e282d5bad24d/ibm_catalog.json#L23C198-L23C216